plugins/eni/commands/commands.go

```diff
@@ -27,6 +30,14 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	ec2InstanceMetadataBackoffMin      = 500 * time.Millisecond
+	ec2InstanceMetadataBackoffMax      = 1 * time.Second
+	ec2InstanceMetadataBackoffMultiple = 2
+	ec2InstanceMetadataBackoffJitter   = 0.2
+	ec2InstanceMetadataTimeout         = 5 * time.Second
```

Comment:
How did you arrive at a 5 second timeout?

Reply:
@aaithal  and I did some tests, the worst case  I saw is 1.5s. So I put 5s here to be safe, but it's changeable.